src/FunctionApps/DSTP/src/main/kotlin/gov/cdc/prime/phdi/Utilities/HL7Parser.kt

```diff
@@ -0,0 +1,117 @@
+package gov.cdc.prime.phdi.Utilities
+
+import ca.uhn.hl7v2.DefaultHapiContext
+import ca.uhn.hl7v2.HL7Exception
+import ca.uhn.hl7v2.HapiContext
+import ca.uhn.hl7v2.model.Message
+import ca.uhn.hl7v2.parser.EncodingNotSupportedException
+import ca.uhn.hl7v2.parser.Parser
+import ca.uhn.hl7v2.validation.ValidationContext;
+import ca.uhn.hl7v2.validation.impl.ValidationContextFactory;
+
+public class HL7Parser() {
+    data class ProcessedMessages(
+        val valid_messages: List<Message>,
+        val invalid_messages: List<String>
+    )
+
+    public fun parse(content: String): ProcessedMessages {
```

Comment:
Is this currently being called?

Reply:
It is not currently being called. It's a placeholder for anticipated future functionality.